M2/Macaulay2/packages/Polyhedra/documentation/old_documentation.m2

```diff
@@ -2557,12 +2557,6 @@ document {
 -- 	  }
 --      }
 
-document {
-     Key => directProduct,
-     Headline => "computes the direct product of two convex objects",
```

Comment:
Also check to see if there are any internal links to `directProduct` that should now be directed to `(directProduct, Cone, Cone)`.